force-app/main/default/classes/SOQL.cls

```diff
@@ -314,18 +367,25 @@ public with sharing class SOQL {
      */
     public static void cloneLineItems(Id origOppId, List<Id> cloneOppIds) {
         // Query the original Opportunity Line Items
-
+        List<OpportunityLineItem> oldOppLineItems = [SELECT Id, PricebookEntryId, UnitPrice FROM OpportunityLineItem WHERE OpportunityId = :origOppId];
         // Create a list to hold the new line items
-
+        List<OpportunityLineItem> newOppLineItems = new List<OpportunityLineItem>();
         // Generate a random number between 1 and 3
-
+        Integer randomNumber = Integer.valueof((Math.random() * 3));
         // Loop through each cloned Opportunity Id
-
             // Loop through each original line item
-
                 // Clone the original line item and add the random number to the quantity
 
+        for(Id clonedOpp : cloneOppIds){
+            for(OpportunityLineItem oldOppLineItem : oldOppLineItems){
+                OpportunityLineItem newOppLineItem = oldOppLineItem.clone(false, false, false, false);
+                newOppLineItem.Quantity = randomNumber;
```

Comment:
I believe the method instructions ask you _add_ the randomNumber to the current quantity:
```suggestion
                newOppLineItem.Quantity =  oldOppLineItem.Quantity + randomNumber;
```